src/test/java/uk/gov/hmcts/ccd/endpoint/std/GlobalSearchEndpointIT.java

```diff
@@ -0,0 +1,188 @@
+        List<Party> list = new ArrayList<>();
+        list.add(party);
+        Party partyTwo = new Party();
+        partyTwo.setAddressLine1("address");
+        partyTwo.setPartyName("name");
+        partyTwo.setDateOfBirth("1999-01-01");
+        partyTwo.setPostCode("EC3M 8AF");
+        partyTwo.setEmailAddress("someone@cgi.com");
+        list.add(partyTwo);
+        searchCriteria.setParties(list);
+        payload.setSearchCriteria(searchCriteria);
+
+        mockMvc.perform(post(GLOBAL_SEARCH_PATH)
+            .contentType(JSON_CONTENT_TYPE)
+            .content(mapper.writeValueAsBytes(payload)))
+            .andExpect(status().is(200))
+            .andReturn();
+    }
+
+    @Test
+    public void shouldThrowValidationErrorsWhenDataInvalid() throws Exception {
+
+        GlobalSearchRequestPayload payload = new GlobalSearchRequestPayload();
+        payload.setMaxReturnRecordCount(10001);
+        payload.setStartRecordNumber(0);
+        payload.setSortCriteria(listOfInvalidSortCriteria);
+        SearchCriteria searchCriteria = new SearchCriteria();
+        searchCriteria.setCcdCaseTypeIds(listOfInvalidFields);
+        searchCriteria.setCcdJurisdictionIds(listOfInvalidFields);
+        searchCriteria.setCaseReferences(listOfInvalidFields);
+        searchCriteria.setStateIds(listOfInvalidFields);
+        Party party = new Party();
+        party.setAddressLine1("address");
+        Party secondParty = new Party();
+        secondParty.setAddressLine1("address");
+        secondParty.setDateOfBirth("1999-13-01");
+        List<Party> list = new ArrayList<>();
+        list.add(secondParty);
+        list.add(party);
+        searchCriteria.setParties(list);
+        payload.setSearchCriteria(searchCriteria);
+
+        MvcResult result = mockMvc.perform(post(GLOBAL_SEARCH_PATH)
+            .contentType(JSON_CONTENT_TYPE)
+            .content(mapper.writeValueAsBytes(payload)))
+            .andExpect(status().is(400))
+            .andReturn();
+
+        String response = result.getResponse().getContentAsString();
+        assertTrue(response.contains(ValidationError.PARTIES_INVALID));
+        assertTrue(response.contains(ValidationError.SORT_BY_INVALID));
+        assertTrue(response.contains(ValidationError.SORT_DIRECTION_INVALID));
+        assertTrue(response.contains(ValidationError.MAX_RECORD_COUNT_INVALID));
+        assertTrue(response.contains(ValidationError.START_RECORD_NUMBER_INVALID));
+        assertTrue(response.contains(ValidationError.JURISDICTION_ID_LENGTH_INVALID));
+        assertTrue(response.contains(ValidationError.STATE_ID_LENGTH_INVALID));
+        assertTrue(response.contains(ValidationError.CASE_TYPE_ID_LENGTH_INVALID));
+        assertTrue(response.contains(ValidationError.DATE_OF_BIRTH_INVALID));
+        assertTrue(response.contains(ValidationError.ARGUMENT_INVALID));
+        assertTrue(response.contains(ValidationError.CASE_REFERENCE_INVALID));
```

Comment:
```suggestion
        MvcResult result = mockMvc.perform(post(GLOBAL_SEARCH_PATH)
            .contentType(JSON_CONTENT_TYPE)
            .content(mapper.writeValueAsBytes(payload)))
            .andExpect(status().is(400))
            .andExpect(jsonPath(DETAILS_FIELD, hasItem(ValidationError.PARTIES_INVALID)))
            .andExpect(jsonPath(DETAILS_FIELD, hasItem(ValidationError.SORT_BY_INVALID)))
            .andExpect(jsonPath(DETAILS_FIELD, hasItem(ValidationError.SORT_DIRECTION_INVALID)))
            .andExpect(jsonPath(DETAILS_FIELD, hasItem(ValidationError.MAX_RECORD_COUNT_INVALID)))
            .andExpect(jsonPath(DETAILS_FIELD, hasItem(ValidationError.START_RECORD_NUMBER_INVALID)))
            .andExpect(jsonPath(DETAILS_FIELD, hasItem(ValidationError.JURISDICTION_ID_LENGTH_INVALID)))
            .andExpect(jsonPath(DETAILS_FIELD, hasItem(ValidationError.STATE_ID_LENGTH_INVALID)))
            .andExpect(jsonPath(DETAILS_FIELD, hasItem(ValidationError.CASE_TYPE_ID_LENGTH_INVALID)))
            .andExpect(jsonPath(DETAILS_FIELD, hasItem(ValidationError.DATE_OF_BIRTH_INVALID)))
            .andExpect(jsonPath(DETAILS_FIELD, hasItem(ValidationError.ARGUMENT_INVALID)))
            .andExpect(jsonPath(DETAILS_FIELD, hasItem(ValidationError.CASE_REFERENCE_INVALID)))
            .andReturn();
```

Reply:
Minor: Assertions like these can be improved a bit by checking the errors appear in the correct place too - here's one way of doing it quite neatly, where `DETAILS_FIELD` is `$.details`